server/embedded/src/org/labkey/embedded/LabKeyServer.java

```diff
@@ -246,6 +252,47 @@ private List<ContextResource> getDataSourceResources(ContextProperties props) th
                 return  dataSourceResources;
             }
 
+            private List<ContextResource> getExtraContextResources(ContextProperties contextProperties) throws ConfigException
+            {
+                List<ContextResource> contextResources = new ArrayList<>();
+                Map<String, Map<String, String>> resourceMaps = Objects.requireNonNullElse(contextProperties.getResources(), Collections.emptyMap());
+
+                for (Map.Entry<String, Map<String, String>> entry : resourceMaps.entrySet())
+                {
+                    Map<String, String> resourceMap = new CaseInsensitiveKeyMap<>();
+                    resourceMap.putAll(entry.getValue());
+                    if (!resourceMap.containsKey("name"))
+                    {
+                        logger.error("Resource configuration error: Ignoring unnamed resource 'context.resources.%s'".formatted(entry.getKey()));
```

Comment:
Some other config errors are fatal on startup, throwing an exception. I think that's probably the right direction to lean unless there's a good reason to be tolerant of an invalid config.